versioned_docs/version-v4.0.0/dapps/mytokentracker.md

```
@@ -0,0 +1,30 @@
+
+1. Visit the MyTokenTracker website: [https://mytokentracker.xyz](https://mytokentracker.xyz).
+
+2. Enter an Ethereum address that holds assets and click "**Tracker Search**."
+
+The tool will display all the assets linked to the entered address on the left side of the screen. As these assets load, it also loads specific asset icons, which can reveal your IP address. This information is displayed on the right side of the screen, exposing a metadata leak.
+
+For example, if you use a DEX like Uniswap, asset logos are loaded by making a request from your computer, making it easy to link your Ethereum address and IP address.
+
+![MyTokenTracker shows data leak](/img/dapps/mytokentracker-leaked-info.png)
+
+3. To prevent your IP address from being exposed, use the HOPR mixnet. Click "**I'm feeling private**" to see how this works. On the right side of the screen, you'll notice that your IP address is no longer visible in the logs.
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)